src/client/src/components/pages/participants/addParticipant/index.jsx

```diff
@@ -25,17 +27,56 @@ export default class index extends Component {
     this.setState(fields);
   };
 
+  addParticipant = obj => {
+    swal({
```

Comment:
If you are interested in trying, this function might be easier to follow if you switch to using `await`.
```
const confirm = await swal(...)
if (confirm.value) {
  const result = await axios(...)
  ...
```

Reply:
thanks @tomduggan85 
I will do it in different branch :smile: